main.cpp

```diff
@@ -474,212 +460,174 @@ static const char *get_conf_dir()
     return DEFAULT_CONF_DIR;
 }
 
-static struct endpoint_config *search_endpoints(const char *name)
+static int parse_mavlink_dialect(const char *val, size_t val_len, void *storage, size_t storage_len)
```

Comment:
I though about creating another struct to map this, but I didn't work on that. We could have this, but we would need a user_data pointer to be used in parser functions, with the mapping. Not sure if it is worth it.